gcp-observability/src/main/java/io/grpc/gcp/observability/GcpObservability.java

```diff
@@ -130,7 +130,7 @@ private void setProducer(
       tracerFactories.add(
           InternalCensusStatsAccessor.getServerStreamTracerFactory(true, true, true));
     }
-    if (config.isEnableCloudTracing()) {
+    if (config.isEnableCloudTrace()) {
```

Comment:
These internal variable/method names don't matter. The only thing that matters is the property name in the JSON file which need to be changed from `enable_cloud_tracing` to `enable_cloud_trace` . That's why this PR has 6 files in it

Reply:
Reverted it back. PTAL